utils/redis_init_script

```diff
@@ -12,8 +12,8 @@
 ### END INIT INFO
 
 REDISPORT=6379
-EXEC=/usr/local/bin/redis-server
-CLIEXEC=/usr/local/bin/redis-cli
+EXEC=/usr/local/bin/placeholderkv-server
+CLIEXEC=/usr/local/bin/placeholderkv-cli
 
 PIDFILE=/var/run/redis_${REDISPORT}.pid
 CONF="/etc/redis/${REDISPORT}.conf"
```

Comment:
We need decisions about these file names. I believe they are used in `make install`.

```suggestion
SERVER_NAME=placeholderkv
PIDFILE=/var/run/${SERVER_NAME}_${PORT}.pid
CONF="/etc/${SERVER_NAME}/${PORT}.conf"
```